core/sail/spin/src/test/resources/functions/apf/concat.sparql

```diff
@@ -0,0 +1,5 @@
+prefix apf: <http://jena.hpl.hp.com/ARQ/property#>
```

Comment:
You should be able to use rdf4j-rio to load .srx as a list of bindings and then compare with the query results.